src/util.js

```diff
@@ -0,0 +1,3 @@
+export default function capitalize(str) {
```

Comment:
I tillegg, så er det fint at det lages tester for alt som lages. Nå skal det sies at jeg kanskje ikke helt ser nytten av å skrive tester for html som komponenter rendrer, og komponenter bør være litt "dumme". Men alle funksjoner, som f.eks. capitalize, bør ha tester. I tillegg bør de være "pure", dvs at de ikke endrer argumentene som kommer inn (slik du har gjort med capitalize).

Testene skal ligge under "test", og bør ha samme struktur som koden i "src", bare at filnavnet i test bør ende med `_spec`, så `src/util.js` sine tester bør ligge i `test/util_spec.js`.

Her er et forslag til hvordan denne kan se ut:
```
import { expect } from 'chai';

import {capitalize} from '../src/util';

describe('util.capitalize', () => {

  it('makes first letter in a string capital', () => {
    const text = 'a random string';
    expect(capitalize(text)).to.equal('A random string');
  });

});
```

Du må gjerne legge til flere tester (med flere `it()`), og dersom det kommer flere funksjoner i util.js, så kan du jo legge til flere `describe()` også.

For å kjøre testene, så går du til kommando-linjen og skriver `npm run test`